src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -324,12 +335,15 @@ const General = ({
           </Select>
         </Item>
 
+        {childProfiles.rfs.length > 0 && (
+          <Item label="RF Profile">{childProfiles.rfs.map(rf => rf.name)}</Item>
+        )}
         <Item label="Summary">
           <Item>
             <Table
               rowKey="id"
               scroll={{ x: 'max-content' }}
-              dataSource={selectedProfile?.childProfiles}
+              dataSource={childProfiles.other}
```

Comment:
instead just filter the childProfiles by type 'ssid' for the table otherwise you'll have to fix this again when another profile type is in the list